### PR TITLE
Navbar component

### PR DESCRIPTION
**Pull request by emilyPhee**
Build navbar component:

- Add logo.
- Add navbar menu items and connect to each page.
- Add LanguageToggle component and functionality.

### CODE REVIEW

File: nextjs-client/src/components/Layout/Navbar/style.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 5rem;
+    height: 5rem;
+
+    font-size: 2.1rem;
+  }
+
+  .language-toggle-wrapper {
```
Comment (anthopark) [inline nextjs-client/src/components/Layout/Navbar/style.js]: Can we remove this?

Comment (emilyPhee) [reply nextjs-client/src/components/Layout/Navbar/style.js]: Yes, I deleted it!

**Review by emilyPhee — COMMENTED** [review]
Reviewed changed files.

**Comment by anthopark** [comment]
[Navbar Component](https://trello.com/c/bk1lmrp6/2-navbar-component)

**Pull request merged**